ambry-router/src/main/java/com.github.ambry.router/BackgroundDeleteRequest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.router;
+
+import com.github.ambry.store.StoreKey;
+
+
+/**
+ * This class contains the request parameters for a background delete operation.
+ */
+class BackgroundDeleteRequest {
+  static final String SERVICE_ID_PREFIX = "ambry-bg-delete-";
```

Comment:
expand to `ambry-background-delete-` if we are not that conservative on string length?